<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_temporal_noise_width">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_width"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_temporal_noise_height">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_height"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>125</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_temporal_noise_bitshift">
      <property name="text">
       <string>BitShift</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_temporal_noise_bitshift"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>143</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_temporal_noise_pixeloffset">
      <property name="text">
       <string>PixelOffset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_temporal_noise_pixeloffset"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>143</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_temporal_noise_inputformat">
      <property name="text">
       <string>InputFormat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_temporal_noise_inputformat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_temporal_noise_outputformat">
      <property name="text">
       <string>OutputFormat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_temporal_noise_outputformat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="label_temporal_noise_dataformat">
      <property name="text">
       <string>DataFormat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_temporal_noise_dataformat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <widget class="QLabel" name="label_temporal_noise_cfa">
      <property name="text">
       <string>CFA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_temporal_noise_cfa"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <item>
     <widget class="QLabel" name="label_temporal_noise_start_x">
      <property name="text">
       <string>start_x</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_start_x"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_20">
    <item>
     <widget class="QLabel" name="label_temporal_noise_start_y">
      <property name="text">
       <string>start_y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_start_y"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>105</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_21">
    <item>
     <widget class="QLabel" name="label_temporal_noise_roi_width">
      <property name="text">
       <string>roi_width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_roi_width"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_22">
    <item>
     <widget class="QLabel" name="label_temporal_noise_roi_height">
      <property name="text">
       <string>roi_height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_roi_height"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_temporal_noise_calculate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_23">
    <item>
     <widget class="QLabel" name="label_temporal_noise_smva1288">
      <property name="text">
       <string>EMVA1288</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_emva1288"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_14">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_24">
    <item>
     <widget class="QLabel" name="label_temporal_noise_smia">
      <property name="text">
       <string>SMIA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_temporal_noise_smia"/>
    </item>
    <item>
     <widget class="QLabel" name="label_temporal_noise_db">
      <property name="text">
       <string>DB</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
